packages/graphback/src/layers/data/InMemoryDataProvider.1.ts

```diff
@@ -0,0 +1,23 @@
+import { GraphbackDataProvider } from './GraphbackDataProvider';
+/**
+ * In Mem data provider used for testing capabilities
+ */
+// TODO
+// tslint:disable-next-line: no-any
+export class InMemoryDataProvider<Type = any, GraphbackContext = any> implements GraphbackDataProvider<Type, GraphbackContext> {
```

Comment:
@b1zzu This is pushed as an example of InMemoryData provider. 
I would need to pick something. From what I see you have been researching SqlLite InMem that might make this implementation obsolete (as knex will work with that as well)

Reply:
If we create a generic Knex we can use SQLite as in-memory database so we don't have to create a separate Provider, but there are also alternative like https://github.com/techfort/LokiJS that we could explor